lib/feature_management.rb

```diff
@@ -48,8 +48,9 @@ def self.enable_identity_verification?
     Figaro.env.enable_identity_verification == 'true'
   end
 
-  def self.reveal_usps_code?
-    Rails.env.development? || current_env_allowed_to_see_usps_code?
+  def self.reveal_usps_flow?
+    env = Rails.env
+    env.development? || env.test? || current_env_allowed_to_see_usps_code?
```

Comment:
Yes, but I'll add a separate method since combing them will defeat the purpose of being able to toggle the feature in production without prefilling the code.